### DISCUSSION TITLE
What version python should I be using?

### DISCUSSION THREAD

**Discussion by jmorri11** [sezanzeb/input-remapper#985] [General]
I'm using Linux Mint, and trying to use input-remapper with a Razer Tartarus v2.  Whenever I go into try and map keys, the program does essentially nothing.  I ran it with debug enabled and got messages about asyncio and locks.  Searching for that lead me to a post about upgrading python and trying again, but there was no further info.  I currently have python v3.10.15 and python3 version 3.8.10.  Is there a higher version I should be using?

**Comment by Tesla-Tank** [comment]
So I also have the razer tartarus v2 but I'm using it on Nobara. I loaded up a Mint live session, and everything seemed to be working (Both manual and ubuntu install). Though I didn't change that much and only changed a few keys. I have python version 3.12.3 installed if this helps.
